lib/awful/layout/suit/tile.lua

```diff
@@ -317,6 +324,9 @@ end
 function tile.left.mouse_resize_handler(c, corner, x, y)
     return mouse_resize_handler(c, corner, x, y, "left")
 end
+function tile.left.skip_gap(nclients, t)
+    return tile.skip_gap(nclients, t)
+end
```

Comment:
However, this is not really important and the current version is "basically okay" as well